examples/http-request-error-interceptor/http_request_error_interceptor.bal

```diff
@@ -0,0 +1,77 @@
+import ballerina/http;
+import ballerina/io;
+
+// Header name checked by the first request interceptor.
+final string interceptor_check_header = "X-requestCheckHeader";
+
+// Header value to be set to the request in the request error interceptor.
+final string interceptor_check_header_value = "RequestErrorInterceptor";
+
+service class RequestInterceptor1 {
+    *http:RequestInterceptor;
+
+    resource function 'default [string... path](http:RequestContext ctx, 
+                            http:Request req) returns http:NextService|error? {
+        io:println("Executing Request Interceptor 1");
+        // Try to read header. This will return a `HeaderNotFoundError` if we do not set this header. Then the execution will 
```

Comment:
```suggestion
        // Tries to read the header. This will return a `HeaderNotFoundError` if you do not set this header. Then, the execution will 
```